rooms_tab.py

```
@@ -0,0 +1,203 @@
+import sys
+from PyQt5.QtCore import Qt
```

Ruff (0.11.9)

`PyQt5.QtCore.Qt` imported but unused

Remove unused import: `PyQt5.QtCore.Qt`

(F401)